internal/spec/templates/configuration.mustache

```diff
@@ -23,27 +24,31 @@ const ApiVersion = "{{.}}"
 {{/withCustomMiddlewareFunction}}
 // Configuration stores the configuration of the API client.
 type Configuration struct {
-Host             string            `json:"host,omitempty"`
-Scheme           string            `json:"scheme,omitempty"`
-DefaultHeader    map[string]string `json:"defaultHeader,omitempty"`
-UserAgent        string            `json:"userAgent,omitempty"`
-Debug            bool              `json:"debug,omitempty"`
-ServerURL        string
-HTTPClient       *http.Client
-apiKey			 string
+Host              string            `json:"host,omitempty"`
+Scheme            string            `json:"scheme,omitempty"`
+DefaultHeader     map[string]string `json:"defaultHeader,omitempty"`
+UserAgent         string            `json:"userAgent,omitempty"`
+Debug             bool              `json:"debug,omitempty"`
+GetRequestRetries int               `json:"getRequestRetries,omitempty"`
+RetryInterval     time.Duration     `json:"retryInterval,omitempty"`
+ServerURL         string
+HTTPClient        *http.Client
+apiKey			  string
 {{#withCustomMiddlewareFunction}}
-    Middleware       MiddlewareFunction
+Middleware        MiddlewareFunction
 {{/withCustomMiddlewareFunction}}
 }
 
 // NewConfiguration returns a new Configuration object.
 func NewConfiguration(apiKey string) *Configuration {
 cfg := &Configuration{
-DefaultHeader:    make(map[string]string),
-UserAgent:        "{{{httpUserAgent}}}{{^httpUserAgent}}ccloud-sdk-go/{{{packageVersion}}}{{/httpUserAgent}}",
-Debug:            false,
-ServerURL:        DefaultServerURL,
-apiKey:			  apiKey,
+DefaultHeader:     make(map[string]string),
+UserAgent:         "{{{httpUserAgent}}}{{^httpUserAgent}}ccloud-sdk-go/{{{packageVersion}}}{{/httpUserAgent}}",
+Debug:             false,
+GetRequestRetries: 0,
```

Comment:
defaulting to off?  I'm good with it, I just wanted to make sure its not a mistake.